modules/ROOT/pages/index.adoc

```diff
@@ -5,75 +5,48 @@ endif::[]
 
 == Need More Information on an Asset?
 
 Click the Helpful Links item in the left navigation area for links to documentation.
 
-== Exchange Content
-
-Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API Groups, and policies), and Runtime Manager (APIs), and you can publish APIs from Design Center to Exchange.
-
-MuleSoft certifies its public content and reviews and approves partner connectors. All private content is internal to an organization and not visible outside that organization. Customers are responsible for any private content they publish within their Exchange instance.
-
 == Language Support
 
 Using multiple languages with Unicode is supported fully throughout Exchange and in all metadata fields, including asset names, descriptions, markdown pages, categories, tags, and reviews.
 
-[[content-types]]
-== Content Types
-
-Both the public and private content in Exchange can consist of the following kinds of elements:
-
-* Connectors - Packaged connectivity to an endpoint developed and deployed on Anypoint Platform with third-party APIs and standard integration protocols
-+
-Use connectors within your application's flows to send and receive data using a protocol or specific API. Anypoint Studio comes with many bundled connectors, and Exchange has many more.
-* Templates - Packaged integration patterns built on best practices to address common use cases
-+
-You can add your information such as user credentials to complete the template's use case or solution, and you can customize or extend templates as needed.
-* Examples - Applications that are ready to run in Anypoint Studio and demonstrate a use case or solution
-* Policies - Configuration modules to extend the functionality of an API and enforce capabilities such as security
-* REST APIs - A RAML file or OAS file that specifies an API
-+
-These APIs can be referenced by an HTTP Request connector to expose metadata to Anypoint Studio.
-* AsyncAPI specifications - An AsyncAPI specification file that specifies an event-driven API
-* SOAP APIs - A WSDL file that specifies an API
-* HTTP APIs - A placeholder for an endpoint for use by private Exchange users who want to manage the endpoint with API Manager
-* API Groups - A set of APIs bundled into a single asset
-+
-Instead of requesting access to multiple APIs to satisfy a use case, a developer can access the group in one step.
-* API Spec Fragments - A RAML document that has a version and an identifier, but is not in itself a complete RAML specification
-+
-API spec fragments are also known as RAML fragments.
-* Custom - A description and an optional file to explain aspects of your system, to provide instructional videos, or to describe product or organizational documentation
 
 == Versions
 
 This page applies to the current version, Exchange 2.
```

Comment:
Please reconcile this with "the API Exchange version 2" (or 1) in other parts of the narrative